INSTALL.md

```diff
@@ -0,0 +1,163 @@
+```
+It may take a long time to wait here. You can use docker to pre-pull the image to solve the problem:
+
+`docker pull kindest/node:v1.23.17`
+
+![image.png](./docs/img/1680576955281-5cf9d091-0cba-422c-9e46-553fadab86fc.png)
+## 1.2 Install kubectl
+```
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+kubectl version
+```
+## 1.3 Install Helm
+```
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+helm version
+```
+# 2. Deploy with Helm
+## 2.1 Install Dragonfly components
+```
+helm repo add dragonfly https://dragonflyoss.github.io/helm-charts/
+helm install --wait --timeout 10m --dependency-update --create-namespace --namespace dragonfly-system dragonfly dragonfly/dragonfly --set dfdaemon.config.download.prefetch=true,seedPeer.config.download.prefetch=true
+```
+It may take a long time to wait here. If the waiting time exceeds 10 minutes, it may be because:
+
+- The server hardware configuration is low.
+
+- The network is too slow to pull images.
+
+![image.png](./docs/img/1680577399074-bc8a9ed5-a37c-4c68-b3fe-fa24bfb4a021.png)
+## 2.2 Install Nydus components
+```
+curl -fsSL -o config-nydus.yaml https://raw.githubusercontent.com/dragonflyoss/Dragonfly2/main/test/testdata/charts/config-nydus.yaml
+
+helm install --wait --timeout 10m --dependency-update --create-namespace --namespace nydus-snapshotter nydus-snapshotter dragonfly/nydus-snapshotter -f config-nydus.yaml
+```
+![image.png](./docs/img/1680577425168-daa8787a-0a54-4300-8ed0-86354e1f11bc.png)
+# 3. Run Nydus image
+## 3.1 Wait for Nydus components to be ready
+```
+pod=`kubectl -n nydus-snapshotter get pods --no-headers -o custom-columns=NAME:metadata.name`
+echo "snapshotter pod name ${pod}"
+# Wait for Nydus components to be ready
+kubectl -n nydus-snapshotter wait po $pod --for=condition=ready --timeout=2m
+```
+![image.png](./docs/img/1680577472445-c9c0b582-f3e6-4cde-9dc3-19f8ed8eb9d2.png)
+## 3.2 Start the pod based on the Nydus image
+```
+cat <<EOF > nydus-pod.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nydus-pod
+spec:
+  containers:
+    - name: nginx
+      image: ghcr.io/dragonflyoss/image-service/nginx:nydus-latest
+      imagePullPolicy: Always
+      command: ["sh", "-c"]
+      args:
+        - tail -f /dev/null
+EOF
+
+kubectl apply -f ./nydus-pod.yaml
+kubectl wait po nydus-pod --for=condition=ready --timeout=1m
+kubectl exec -ti nydus-pod  -- date
+```
+![image.png](./docs/img/1680577518512-bfbe9818-a475-4754-9090-e52713505984.png)
+If the date is successfully output, it means that the nydus pod started successfully.
+## 3.3 Convert Nydus image (optional)
+In the above section we used a pre-converted Nydus image: `ghcr.io/dragonflyoss/image-service/nginx:nydus-latest`
+
+You can also use Nydusify to convert new Nydus images from OCI images.
```

Comment:
Maybe give a link to `https://github.com/dragonflyoss/image-service/blob/master/docs/nydusify.md`.

Reply:
Fixed.